app/scrapers/sql/courses.sql

```diff
@@ -0,0 +1,81 @@
+CREATE TABLE IF NOT EXISTS courses (
+    id SERIAL PRIMARY KEY,
+    course_id text NOT NULL,
+    semester int NOT NULL,
+    name text NOT NULL,
+    dept_id text NOT NULL,
+    department text NOT NULL,
+    credits text NOT NULL,
+    description text,
+    grading_method text[],
+    gen_ed text[],
+    relationships jsonb,
+    UNIQUE(course_id, semester)
+);
+CREATE INDEX on courses(semester);
+
+CREATE TABLE IF NOT EXISTS sections (
+    id SERIAL PRIMARY KEY,
+    section_id text NOT NULL,
+    course_id text NOT NULL,
+    semester int NOT NULL,
+    number text NOT NULL,
+    seats text NOT NULL,
+    meetings jsonb NOT NULL,
+    open_seats text NOT NULL,
+    waitlist text,
+    instructors text[],
```

Comment:
I'd like to be able to remove this and query directly; I'll leave it as a TODO though.